cfme/tests/containers/test_labels.py

```diff
@@ -0,0 +1,230 @@
+    pods_all_names = [x[0] for x in pods_list]
+    pods_names_revised = filter(lambda ch: 'metrics-deployer' not in ch, pods_all_names)
+    pod_name = str(random.choice(pods_names_revised))
+    pod_name_in_project = [x.project_name for x in provider.mgmt.list_container_group()
+                           if x[0] == pod_name][0]
+    # pod endpoint
+    pod_endpoint = 'https://' + provider.hostname + ':8443/api/v1/namespaces/' + \
+                   pod_name_in_project + '/pods/' + pod_name
+
+    # picking the master node
+    node_name = provider.hostname
+    # node endpoint
+    node_endpoint = 'https://' + provider.hostname + ':8443/api/v1/nodes/' + provider.hostname
+    # template name will be picked randomly
+    template_name = str(random.choice(provider.mgmt.list_template())[0])
+    template_name_in_project = [
+        x.project_name for x in provider.mgmt.list_template() if x[0] == template_name][0]
+    # template endpoint
+    template_endpoint = 'https://' + provider.hostname + ':8443/oapi/v1/namespaces/' + \
+                        template_name_in_project + '/templates/' + template_name
+    # image name will be picked randomly
+    image_name = str(random.choice(provider.mgmt.list_image_openshift())['name'])
+    # image endpoint
+    image_endpoint = 'https://' + provider.hostname + ':8443/oapi/v1/images/' + image_name
+
+    object_names.extend([project_name, service_name, rc_name, pod_name, node_name,
+                         template_name, image_name])
+
+    obj_endpoints.extend([project_endpoint, service_endpoint, rc_endpoint, pod_endpoint,
+                          node_endpoint, template_endpoint, image_endpoint])
+
+    # use different label for each object and generate json
+    for obj_name in objects_ocp:
+        key_label = fauxfactory.gen_alphanumeric(6)
+        value_label = fauxfactory.gen_alphanumeric(6)
+        json_payload_format = '{"apiVersion":"v1","kind":' + obj_name + \
+                              ',"metadata":{"labels":{' + '"{}"'.format(key_label) + ':' \
+                              + '"{}"'.format(value_label) + '}}}'
+        json_new_payload.append(json_payload_format)
+
+    return json_new_payload, object_names, obj_endpoints
+
+
+def check_request_status(provider):
+    json_api_payload, obj_api_names, obj_api_endpoints = create_api_data(provider)
+
+    endpoints_json_dict = dict(zip(obj_api_endpoints, json_api_payload))
+
+    headers = {'Authorization': 'Bearer' + ' ' + provider.mgmt.auth,
+               'Content-Type': 'application/strategic-merge-patch+json'}
+
+    for key, value in endpoints_json_dict.iteritems():
+        req = requests.patch(key, headers=headers, verify=False, data=value)
+        assert req.status_code == 200
+
+    new_lbls_list = []
+
+    for index in range(len(json_api_payload)):
+        obj = (json.loads(json_api_payload[index])).values()[2]
+        new_lbls_list.append(obj)
+
+    # create list of keys and list of values
```

Comment:
@gshefer Could you elaborate? Cause I am parsing JSON output and that output is different for each object in the list.